homeassistant/helpers/entity_platform.py

```diff
@@ -621,8 +621,13 @@ async def _async_add_entity(  # noqa: C901
                         **device_info,
                     )
                 except dev_reg.DeviceInfoError as exc:
-                    self.logger.error("Ignoring invalid device info: %s", str(exc))
-                    device = None
+                    self.logger.exception(
```

Comment:
Why log this as `exception`? We don't need to print a stack trace as we know where the error is raised from ?

Reply:
Right, maybe it's enough with the name of the platform, I'll change it back 👍